.rubocop_fixme.yml

```diff
@@ -23,7 +23,7 @@ Metrics/AbcSize:
 
 Metrics/ModuleLength:
   Exclude:
-    - 'app/controllers/concerns/hyrax/collections_controller_behavior.rb'
+    - 'app/controllers/concerns/hyrax/dashboard/collections_controller_behavior.rb'
```

Comment:
@ggeisler I see your point about the term dashboard.  I'm wondering if this should be reviewed as part of a larger refactoring of the dashboard/admin features.  I would not want to hold up this PR for something that is a larger discussion and potentially work for another sprint.